app/views/overview/index.html.erb

```diff
@@ -1,13 +1,15 @@
 <% unless @current_org_signed_mou %>
-<div class="govuk-grid-row">
-  <div class="govuk-warning-text" id="mou-warning">
-    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
-    <strong class="govuk-warning-text__text">
-      <span class="govuk-warning-text__assistive">Warning</span>
-      You must sign the GovWifi <%= link_to "memorandum of understanding", mou_index_path, class: "govuk-link govuk-link--no-visited-state" %> to use this service.
-    </strong>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
```

Comment:
@freesteph Which guideline does this relate to?

Reply:
I'm not sure, I just fixed it as a I was tidying up the page and it looked bad on mobile.